src/models/biddingDao.js

```diff
@@ -0,0 +1,98 @@
+const { appDataSource } = require("./appDataSource");
+
+const createBid = async (userId, productId, price) => {
+  return await appDataSource.query(
+    `INSERT INTO biddings (
+      user_id,
+      product_id,
+      price
+    ) VALUES (
+      ?,
+      ${productId},
+      ${price}
+    )`,
+    [userId]
+  );
+};
```

Comment:
```suggestion
const createBid = async (userId, productId, price) => {
  return appDataSource.query(
    `INSERT INTO biddings (
      user_id,
      product_id,
      price
    ) VALUES (
      ?,
      ?,
      ?
    )`,
    [userId, productId, price]
  );
};
```

1. await를 쓰지 않아도 되는 코드입니다. (함수내 다음에 실행할 코드가 없음)
2. placeholder (?) 를 사용하는 방법을 통일해주세요!
3. 2번의 내용은 아래의 다른 dao method들도 똑같이 적용 부탁드릴게요